README.md

```diff
@@ -1,30 +1,72 @@
 siddhi-map-wso2event
 ======================================
 
-This extension,WSO2 Event mapper, is used to convert WSO2
-events to/from Siddhi events. 
+The **siddhi-map-wso2event extension** is an extension to <a target="_blank" href="https://wso2.github.io/siddhi">Siddhi
+</a> that  can be used to convert WSO2 events to/from Siddhi events.
 
-Features Supported
-------------------
-  - WSO2 Event source mapper :Source mapping allows the user to convert WSO2Events to SiddhiEvents.
-  - WSO2 Event sink mapper : Sink mapping allows the user to convert SiddhiEvents to WSO2Events.
+Find some useful links below:
 
-#### Prerequisites for using the feature
-  - Siddhi Stream should be defined
-  - Event Source type should be WSO2 event receiver type
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-wso2event">Source code</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-wso2event/releases">Releases</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-wso2event/issues">Issue tracker</a>
 
-#### Deploying the feature
-Feature can be deployed as an OSGI bundle by putting the jar file of the component to SP_HOME/lib directory of SP 4.0.0 pack. 
+## Latest API Docs 
 
-## How to Contribute
-  * Please report issues at [Siddhi Github Issue Tacker](https://github.com/wso2-extensions/siddhi-map-wso2event/issues)
-* Send your bug fixes pull requests to [master branch](https://github.com/wso2-extensions/siddhi-map-wso2event/tree/master) 
+Latest API Docs is <a target="_blank" href="https://wso2-extensions.github.io/siddhi-map-wso2event/api/4.0.2-SNAPSHOT">4.0.2-SNAPSHOT</a>.
+
+## How to use 
+
+**Using the extension in <a target="_blank" href="https://github.com/wso2/product-sp">WSO2 Stream Processor</a>**
+
+* You can use this extension in the latest <a target="_blank" href="https://github.com/wso2/product-sp/releases">WSO2 Stream Processor</a> that is a part of <a target="_blank" href="http://wso2.com/analytics?utm_source=gitanalytics&utm_campaign=gitanalytics_Jul17">WSO2 Analytics</a> offering, with editor, debugger and simulation support. 
+
+* This extension is shipped by default with WSO2 Stream Processor, if you wish to use an alternative version of this extension you can replace the component <a target="_blank" href="https://github.com/wso2-extensions/siddhi-map-wso2event/releases">jar</a> that can be found in the `<STREAM_PROCESSOR_HOME>/lib` directory.
+
+**Using the extension as a <a target="_blank" href="https://wso2.github.io/siddhi/documentation/running-as-a-java-library">java library</a>**
+
+* This extension can be added as a maven dependency along with other Siddhi dependencies to your project.
+
+```
+     <dependency>
+        <groupId>org.wso2.extension.siddhi.map.wso2event</groupId>
+        <artifactId>siddhi-map-wso2event-parent</artifactId>
```

Comment:
We should use the components artifact ID here

Reply:
Fixed in 7c7e238